ajax/timer.php

```diff
@@ -12,79 +12,128 @@
       case 'start':
          if (PluginActualtimeTask::checkTimerActive($task_id)) {
             $result=[
-               'mensage'=>__("A user is already performing the task", 'actualtime'),
-               'title' => __s('Warning'),
-               'class' => 'warn_msg',
+               'mensage' => __("A user is already performing the task", 'actualtime'),
+               'title'   => __('Warning'),
+               'class'   => 'warn_msg',
             ];
          } else {
-            if (PluginActualtimeTask::checkUserFree(Session::getLoginUserID())) {
+            if (! PluginActualtimeTask::checkUserFree(Session::getLoginUserID())) {
                $opcional=PluginActualtimeTask::getTicket(Session::getLoginUserID());
                $result=[
-                  'mensage'=>__("You are already doing a task", 'actualtime')." <a href='/front/ticket.form.php?id=".$opcional."'>".__("Ticket")."</a>",
-                  'title' => __s('Warning'),
-                  'class' => 'warn_msg',
+                  'mensage' => __("You are already doing a task", 'actualtime')." <a href='/front/ticket.form.php?id=".$opcional."'>".__("Ticket")."&forcetab=Ticket%241</a>",
```

Comment:
What is this for?
If you wanted to open the task tab, is not working

Reply:
I already have this part ready to open a modal window.
![modaltask](https://user-images.githubusercontent.com/39123808/53820215-98da3a80-3f6b-11e9-9670-32fba5c97170.png)

Reply:
Yes, I am not introducing a new modal windows, just a modification to the link. Unfortunately, I inserted the modification in link text, instead of href attibute. Sorry for that.

The intention here is, if we are in any page and receive the information we have an ongoing task, if we click the link, we go directly to Tasks Tab. Otherwise, we would be directed to the last opened tab in Ticket page. I thought it was not the intent. For example, if I have the timer on in Ticket 3, but I am in page of Problems, in Ticket 2, I will receive the notice I have the timer on (Ticket 3). If I click the link, I will got to Ticket 3 page, but still in Problems tab, when I think the goal would be going to the Tasks tab. The same if I go to a computer page, if I click on the timer notice, it seems it should bring me to the task tab of current timer's ticket.

Here we use the parameter as Glpi in its home page, when you click in to do tasks: &forcetab=Ticket$1.

Please, let me know if current behaviour is by design. Sorry for messing with that withour prior discussion.

Corrected the position of the url parameter with commit https://github.com/Multiconecta/actualtime/commit/6900d32cad0bcff67b0e6fa25671834809b55721